### PR TITLE
fix(norwegian_locale): change the i18n locale.name for nb-NO

### LINKED ISSUE #559: Language selector shows only "Bokmål", not "Norsk" or "Norsk Bokmål"
"Bokmål" is a form of Norwegian but I think it's incorrect to show it without "Norsk bokmål" or some variant that starts with the main language.

Wikipedia for reference:
http://puu.sh/H70qY/e369f3cf47.png

Vue:
http://puu.sh/H70rz/499dd97a04.png

For ease of copying
```
Norsk bokmål
Norsk nynorsk
```

~Other alternatives~ doesn't sort well
```
Norsk
Nynorsk
```

### PR DESCRIPTION
**Pull request by AlfHou**
Change the locale name of nb-NO from 'Bokmål' to 'Norsk'

Fixes #559

### CODE REVIEW

**Comment by ferferga** [comment]
Shouldn't we add nynorsk as indicated in the linked issue?

We might not be able to translate it, but somebody can take the translations over in Weblate.

File: nuxt.config.ts [hunk 1]
```diff
@@ -155,7 +155,7 @@ const config: NuxtConfig = {
       { code: 'en', iso: 'en-US', name: 'English', file: 'en-US.json' },
       { code: 'es', iso: 'es-ES', name: 'Español (España)', file: 'es.json' },
       { code: 'fr', iso: 'fr-FR', name: 'Français', file: 'fr-FR.json' },
-      { code: 'nb', iso: 'nb-NO', name: 'Bokmål', file: 'nb_NO.json' },
+      { code: 'nb', iso: 'nb-NO', name: 'Norsk', file: 'nb_NO.json' },
```
Comment (MrTimscampi) [inline nuxt.config.ts:158]:
```suggestion
      { code: 'nb', iso: 'nb-NO', name: 'Norsk bokmål', file: 'nb_NO.json' },
```

This is the proper name for it. As there are two variants of "Norsk", we should explicitly mention which one it is.

**Review by MrTimscampi — CHANGES_REQUESTED** [review]

**Comment by MrTimscampi** [comment]
> Shouldn't we add nynorsk as indicated in the linked issue?

We'd need to wait until someone on Weblate translated it, then we add it. It's pretty useless to add an empty language just for the sake of it.

**Review by MrTimscampi — APPROVED** [review]
Saw the discussion about this on the other issue, so it's all good. We can change it if someone comes around and starts doing a Nynorsk translation.

**Comment by hawken93** [comment]
Nice :+1:

**Pull request merged**